scripts/Test-Signatures.ps1

```diff
@@ -0,0 +1,209 @@
+[CmdletBinding()]
+param (
+    [Parameter()]
+    [String]
+    $Path = "$PSScriptRoot\packages",
+
+    [Parameter()]
+    [Switch]
+    $CIBuild
+)
+
+Add-Type -AssemblyName System.IO.Compression.FileSystem
+
+# Allowed certificate subjects
+$global:ValidSubjects = @( `

```

Comment:
Global is rarely needed. You should pass all your values by parameters, and assign them from output. If you really need to scope value outside of function use $script scope.